app/scripts/services/acceptanceTest.js

```diff
@@ -106,11 +106,9 @@ angular.module('ludwig').factory('AcceptanceTestsService', function($q, $http, $
 
         launchTest: function(test) {
             test.running = true;
+
+            // Remove the test status during execution, so that the top bar of the test accordion is grey.
             delete test.status;
-            test.expectedResults.forEach(function(expectedResult) {
```

Comment:
@MattiSG, do you think this piece of code was actually needed ? 
It seems like `test.expectedResults` is anyway reset in `formatValues`, called by `handleResult` later on `launchTest`.
So my conclusion is that the only point was to have a specific behavior during the execution. But to this end, deleting test.status seems to be enough.
What do you think ?